src/psfmachine/superstamp.py

```diff
@@ -0,0 +1,533 @@
+                lc["flux_err_VA"] = (self.werrs_va[:, idx]) * u.electron / u.second
+            if sap:
+                lc["flux_sap"] = (self.sap_flux[:, idx]) * u.electron / u.second
+                lc["flux_err_sap"] = (self.sap_flux_err[:, idx]) * u.electron / u.second
+
+            self.lcs.append(lc)
+        self.lcs = lk.LightCurveCollection(self.lcs)
+        return
+
+    def plot_image_interactive(self, ax=None, sources=False):
+        """
+        Function to plot the Full Frame Image and the Gaia Sources
+        Parameters
+        ----------
+        ax : matplotlib.axes
+            Matlotlib axis can be provided, if not one will be created and returned
+        sources : boolean
+            Whether to overplot or not the source catalog
+        Returns
+        -------
+        ax : matplotlib.axes
+            Matlotlib axis with the figure
+        """
+        if ax is None:
+            fig, ax = plt.subplots(1, figsize=(10, 10))
+
+        ax = plt.subplot(projection=self.wcs)
+        row_2d, col_2d = np.mgrid[
+            self.row.min() : self.row.max() + 1,
+            self.column.min() : self.column.max() + 1,
+        ]
+        im = ax.pcolormesh(
+            col_2d,
+            row_2d,
+            self.flux_2d[0],
+            cmap=plt.cm.viridis,
+            shading="nearest",
+            norm=colors.SymLogNorm(linthresh=200, vmin=0, vmax=2000, base=10),
+            rasterized=True,
+        )
+        plt.colorbar(im, ax=ax, label=r"Flux ($e^{-}s^{-1}$)", fraction=0.042)
+
+        ax.set_xlabel("R.A. [hh:mm]")
+        ax.set_ylabel("Decl. [deg]")
+        ax.set_xlim(self.column.min() - 2, self.column.max() + 2)
+        ax.set_ylim(self.row.min() - 2, self.row.max() + 2)
+
+        ax.set_aspect("equal", adjustable="box")
+
+        def update(t):
+            ax.pcolormesh(
+                col_2d,
+                row_2d,
+                self.flux_2d[t],
+                cmap=plt.cm.viridis,
+                shading="nearest",
+                norm=colors.SymLogNorm(linthresh=200, vmin=0, vmax=2000, base=10),
+                rasterized=True,
+            )
+            ax.set_title(
+                "%s FFI Ch/CCD %s MJD %f"
+                % (self.meta["MISSION"], self.meta["EXTENSION"], self.time[t])
+            )
+            if sources:
+                ax.scatter(
+                    self.sources.column,
+                    self.sources.row,
+                    facecolors="none",
+                    edgecolors="r",
+                    linewidths=0.5 if self.sources.shape[0] > 1000 else 1,
+                    alpha=0.9,
+                )
+            ax.grid(True, which="major", axis="both", ls="-", color="w", alpha=0.7)
+            fig.canvas.draw_idle()
+
+        interact(update, t=(0, self.flux_2d.shape[0] - 1, 1))
+
+        return
+
+
+def _load_file(fname):
+    """
+    Helper function to load FFI files and parse data.
```

Comment:
I think these are specifically the superstamp files only, let's include a description of the expected file format (e.g. one image per file, n files, etc)